wp-content/themes/humanity-theme/patterns/web-report.php

```diff
@@ -0,0 +1,353 @@
+<?php
+/**
+ * Title: Web Report
+ * Description: A full post pattern for a Web Report
+ * Slug: amnesty/web-report
+ * Keywords: full, web report
+ * Categories: humanity
+ */
+?>
+
+<!-- wp:amnesty-core/block-hero /-->
```

Comment:
We have the new hero block on `develop` now for FSE, so we can probably update this, since they'll be going out at the same time:
```suggestion
<!-- wp:amnesty-core/hero {"content":"Web Report","align":"left"} /-->
```
```suggestion
<!-- wp:amnesty-core/block-hero /-->
```

Reply:
resolved